### PR TITLE
export earnings, txid, token name/value, payee location

### PR DESCRIPTION
**Pull request by owocki**
##### Description

export earnings, txid, token name/value, payee location in earnings export

##### Refers/Fixes

customer request

##### Testing

tested locally

### CODE REVIEW

File: app/marketing/views.py [hunk 1]
```diff
@@ -587,18 +587,24 @@ def account_settings(request):
             response['Content-Disposition'] = f'attachment; filename="{name}.csv"'
 
             writer = csv.writer(response)
-            writer.writerow(['id', 'date', 'From', 'To', 'Type', 'Value In USD', 'url'])
+            writer.writerow(['id', 'date', 'From', 'From Location', 'To', 'To Location', 'Type', 'Value In USD', 'url', 'txid', 'token_name', 'token_value'])
             profile = request.user.profile
             earnings = profile.earnings if export_type == 'earnings' else profile.sent_earnings
-            earnings = earnings.all().order_by('-created_on')
+            earnings = earnings.filter(network='mainnet').order_by('-created_on')
```
Comment (octavioamu) [inline app/marketing/views.py:593]: shouldn't this be dynamic to work on local also ?

Comment (thelostone-mc) [reply app/marketing/views.py:593]: nah it's just for export -> should be fine

**Comment by thelostone-mc** [comment]
@owocki CI fail :|

**Comment by owocki** [comment]
@thelostone-mc just deployed a fix for tests

File: app/dashboard/models.py [hunk 2]
```diff
@@ -1759,10 +1759,13 @@ def psave_tip(sender, instance, **kwargs):
 def postsave_tip(sender, instance, created, **kwargs):
     is_valid = instance.sender_profile != instance.recipient_profile and instance.txid
     if instance.pk and is_valid:
+        value_true = 0
+        value_true = 0
```
Comment (thelostone-mc) [inline app/dashboard/models.py:1763]: ^ @owocki `value_usd` :P 
I'll fix it up and see

**Pull request merged**